veocean-data-farming/df-intro.md

```diff
@@ -31,7 +31,9 @@ All earnings for veOCEAN holders are claimable on the ”Rewards” page inside
 Data assets for DF may published in any [network where Ocean’s deployed in production](../core-concepts/networks.md): Eth mainnet, Polygon, etc.
 
 ### When to claim?
-There are fresh rewards available every Thursday. If you wish, you can wait for many weeks to accumulate before claiming. (It’s all on-chain.)
+There are fresh rewards available every Thursday at midnight GMT. If you wish, you can wait for many weeks to accumulate before claiming. (It’s all on-chain.)
+
+Rewards are calculated based on different factors, like your carrying veOCEAN balance and your Data Farming allocation. You can learn more about these details in places like [the Reward Function](df-background#reward-function) and by learning more about veCRV in the next section.
```

Comment:
can we also add more details about when exactly they can claim relative to the moment when they lock?

I guess users doesn't want to read code in order to understand it, also they might be looking for a text that clearly confirms that.

Reply:
Yeah, that's a good point. Some people have asked than on Discord so makes sense to add it here in the docs

Reply:
I added some info about rewards distribution